packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1030,4 +1036,4 @@ class _Editable extends LeafRenderObjectWidget {
       ..cursorWidth = cursorWidth
       ..cursorRadius = cursorRadius;
   }
-}
+}
```

Comment:
nit: leave the last newline

Reply:
I took it out because it causes the tests to fail.